packages/clerk-js/package.json

```diff
@@ -27,7 +27,9 @@
   "types": "dist/types/index.d.ts",
   "files": [
     "dist",
-    "headless"
+    "headless",
+    "!dist/index.html",
```

Comment:
This ensures that we don't publish these files to npm.

Reply:
can we omit to `/dist/preview/` to ensure we don't accidentally add additional files here and publish them?

```
/dist/preview/index.html
/dist/preview/sandbox.js
```

Reply:
I think that'd be complicated given how Rspack makes decisions based on the emit path. As an alternative, how about we only emit these assets when we run `pnpm build:sandbox`?

Reply:
yeah doing that feels better [17d1d90](https://github.com/clerk/javascript/pull/4608/commits/17d1d906aad4f296e6d7d1805c471d72e64542e4)